### PR TITLE
readability improvements

### PR DESCRIPTION
**Pull request by mstormi**
Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -119,31 +119,31 @@ You can find specific installation instructions for these and other platforms in
 If you have a strong preference towards a particular platform, then that platform is probably your best choice.
 
 You can install openHAB on your desktop computer for evaluation purposes if you already have any of these systems available for use, but we recommend using a dedicated system in the long run. 
-If you feel serious about home automation it may be better to start with a dedicated system right away.
+If you feel serious about home automation it is better to start with a dedicated system right away.
 
 If you have no strong preference, get a [Raspberry Pi 4](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) with 2 GB and a 16 GB SD card and install [openHABian]({{base}}/installation/openhabian.html) for the best experience.
-A RPi 2 or 3 is fine, too, but a RPi4/8GB is overkill, as are larger SD cards.
+A RPi 2 or 3 is fine, too, but a RPi 4 with 8GB is overkill, as are larger SD cards.
```
Comment (ghys) [inline introduction.md:125]:
```suggestion
A RPi 2 or 3 is fine, too, but a RPi 4 with 8GB of RAM is overkill, as are larger SD cards.
```

File: introduction.md [hunk 2]
```diff
@@ -119,31 +119,31 @@ You can find specific installation instructions for these and other platforms in
...
 
-While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system easily capable of automating your entire home.
-It is worth noting two potential limitations of Raspberry Pis, however:
-RPi 3 and older are limited in RAM (1 GB of memory or less) and may not perform well when additional memory hungry applications such as databases and data visualization programs are being used.
-Two or more GB should be fine.
+While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB eco system in a box and easily capable of automating your entire home.
+Mind two potential limitations of Raspberry Pis:
```
Comment (ghys) [inline introduction.md:128]:
```suggestion
Please be aware of two potential limitations of Raspberry Pis:
```

File: introduction.md [hunk 3]
```diff
@@ -119,31 +119,31 @@ You can find specific installation instructions for these and other platforms in
...
 
 If you have no strong preference, get a [Raspberry Pi 4](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) with 2 GB and a 16 GB SD card and install [openHABian]({{base}}/installation/openhabian.html) for the best experience.
-A RPi 2 or 3 is fine, too, but a RPi4/8GB is overkill, as are larger SD cards.
+A RPi 2 or 3 is fine, too, but a RPi 4 with 8GB is overkill, as are larger SD cards.
 
-While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system easily capable of automating your entire home.
-It is worth noting two potential limitations of Raspberry Pis, however:
-RPi 3 and older are limited in RAM (1 GB of memory or less) and may not perform well when additional memory hungry applications such as databases and data visualization programs are being used.
-Two or more GB should be fine.
+While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB eco system in a box and easily capable of automating your entire home.
```
Comment (ghys) [inline introduction.md:127]:
```suggestion
While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB ecosystem in a box and easily capable of automating your entire home.
```

File: introduction.md [hunk 4]
```diff
@@ -119,31 +119,31 @@ You can find specific installation instructions for these and other platforms in
...
 
 You will quickly discover that you may want to learn more about Things, Channels, Items, and more. 
 To do so, we highly recommend that you read the next chapter titled [Concepts]({{base}}/concepts/index.html). 
 It provides more in-depth descriptions of Things, Items, Bindings, etc. that will help you as you dive deeper into openHAB.
 
-The amount of information provided here can be overwhelming, so please come back to these sections often as you develop your home automation system.
+The amount of information provided can be overwhelming but don't hesitate to return here to lookup infos.
 
-Along the way, you may have some questions; the openHAB community is here to help.
+When you encounter questions along the way that the docs don't answer for you, the openHAB community is there to help.
```
Comment (ghys) [inline introduction.md:142]:
```suggestion
When you encounter questions along the way that the docs don't answer for you, the openHAB community is here to help.
```

**Review by ghys — COMMENTED** [review]
A few suggestions:

**Review by Confectrician — APPROVED** [review]
Thanks. 👍

**Pull request merged**